ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -555,6 +555,29 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8
+- as: aws-ipi-efa-pg-f14
```

Comment:
How about f28? Considering we also added a destructive testing and the job need a big instance type, maybe we can save some budget for it.

Reply:
done